models/export.js

```diff
@@ -0,0 +1,58 @@
+
+
+Meteor.methods({
+  exportBoard(boardId) {
+    check(boardId, String);
+    const board = Boards.findOne(boardId);
+    if(board.isVisibleByUser()) {
+      const exporter = new Exporter(boardId);
+      return exporter.build();
+    } else {
+      throw new Meteor.Error('error-board-notAMember');
+    }
+  },
+});
+
+class Exporter {
+  constructor(boardId) {
+    this._boardId = boardId;
+  }
+
+  build() {
+    const byBoard = {boardId: this._boardId};
```

Comment:
added a "_format" field, set to "wekan-board-1.0.0" to be as self-descriptive as possible